init-tools.ps1

```diff
@@ -57,6 +57,61 @@ if (!(Test-Path "$BUILD_TOOLS_PACKAGE_PATH\init-tools.cmd"))
     exit 1
 }
 
+# Bring down the CLI for build tools
+$DOTNET_PATH=$BUILD_TOOLS_PATH + "\dotnetcli"
+
+Write-Host "Installing Build Tools CLI version..."
+if (!(Test-Path "$DOTNET_PATH"))
+{
+     mkdir "$DOTNET_PATH"
+}
+
+$DOTNET_VERSION = Get-Content "$RepoRoot\BuildToolsCLIVersion.txt"
+$DOTNET_ZIP_NAME="dotnet-dev-win-x64." + $DOTNET_VERSION + ".zip"
```

Comment:
What you're doing here is more hoops than copy/pasting the two liner I linked to :)

What is here as-is is immediately stale code, that has to be kept in sync with the implementation of the install scripts. When we change to a preview3 version the switch could be as easy as changing which install script is being used, or as difficult as figuring out the new storage layout the newer install scripts use and replicating the logic again.